src/mozanalysis/bootstrap.py

```diff
@@ -52,7 +53,7 @@ def bootstrap(sc, data, stat_fn, num_iterations=2000, confidence_level=0.95):
 
     broadcast_data.unpersist()
 
-    p = _percentile(data, confidence_level)
+    p = _percentile(stats, confidence_level)
```

Comment:
That seems unrelated?

Reply:
Yes, it's now in another PR and this branch will be rebased on that when it is merged. You have eagle eyes. :eagle: :eyes: